armi/physics/neutronics/crossSectionGroupManager.py

```diff
@@ -1002,7 +1015,23 @@ def _setBuGroupBounds(self, upperBuGroupBounds):
                 raise ValueError("Burnup groups must be ascending")
             lastBu = upperBu
 
-    def _updateBurnupGroups(self, blockList):
+        self._buGroupBounds = buGroupBounds + [float("inf")]
+
+    def _setTempGroupBounds(self, tempGroupBounds):
+        """Set the temperature group structure."""
+        lastTemp = -C_TO_K
+        # validate structure
+        for upperTemp in tempGroupBounds:
+            if upperTemp < -C_TO_K:
+                raise ValueError(
+                    "Temperature boundary is below absolute zero {0}.format(upperTemp)"
+                )
+            if upperTemp < lastTemp:
+                raise ValueError("Temp groups must be ascending")
+            lastTemp = upperTemp
+        self._tempGroupBounds = tempGroupBounds + [float("inf")]
+
+    def _updateEnvironmentGroups(self, blockList):
```

Comment:
Update docstring to discuss temperature groups in addition to burnup groups

Reply:
done